Align "workgroup" spelling

alexandersandberg opened the pull request:
This fixes #89.

### Motivation:

As discussed among SWWG, we decided that it would be a good idea to align the spelling of "workgroup" according to the [Apple Style Guide](https://help.apple.com/applestyleguide/). This is the first step in a plan to introduce a style guide for Swift.org content.

### Modifications:

- Change any occurrences of "work group(s)" to "workgroup(s)"...
  - in static website content
  - in two recent blog posts
  - in `_data/authors.yml` for Diversity in Swift workgroup

### Discussions:

**Should we also update the spelling of "workgroup" in older blog posts?**

**Should we also update the spelling of "workgroup" in personal author bios in `_data/authors.yml`?**

alexandersandberg commented:
@swift-ci test

tomerd commented:
cc @tkremenek

shahmishal commented on diversity/index.md:
```diff
@@ -22,7 +22,7 @@ The current Diversity in Swift work group consists of the following people:
 * Timirah James ([@timirahj](https://forums.swift.org/u/timirahj))
 
-You can reach the Diversity in Swift work group on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-work-group).
+You can reach the Diversity in Swift workgroup on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-work-group).
```
```suggestion
You can reach the Diversity in Swift workgroup on the Swift Forums as [@diversity-work-group](https://forums.swift.org/g/diversity-workgroup).
```

Last week I updated forums.swift.org to workgroup from work-group.

shahmishal replied:
Also, we should update line 31.

alexandersandberg replied:
Oh, I was not aware of this change. Thanks!

https://forums.swift.org/g/diversity-workgroup seems to be private?

shahmishal commented on _data/authors.yml:
```diff
@@ -259,8 +259,8 @@ parkera:
    about: "Tony Parker manages teams at Apple working on Foundation and Swift packages."
 
 diversity-work-group:
```
```suggestion
diversity-workgroup:
```
We should also update the blog post which contains this reference

shahmishal replied:
- _posts/2020-12-16-accessibility-and-inclusion.md
- _posts/2020-12-16-diversity-in-swift.md
- _posts/2021-05-10-swift-mentorship-program.md

shahmishal commented:
Few more files we should update the text: 
- gsoc2022/index.md
   - Line 397
- mentorship-tos/index.md
   - Line 32
- mentorship/index.md 
   - Line 38, 98 and 104 


@alexandersandberg Thank you for getting the website updated to "workgroup"!

alexandersandberg commented:
Thank you, @shahmishal!

I have updated the diversity workgroup and interop workgroup references in e83f00a and 7db3888.

I also added a link to one of the diversity workgroup references in c382fcd.

shahmishal commented:
> Should we also update the spelling of "workgroup" in older blog posts?

It would be good to figure out how many blog posts we are talking about.

> Should we also update the spelling of "workgroup" in personal author bios in _data/authors.yml?

We should and `cc` the authors make them aware of the change.

alexandersandberg commented:
@shahmishal 

> It would be good to figure out how many blog posts we are talking about.

Only 3 blog posts would need to be updated:
- `_posts/2016-10-25-server-api-workgroup.md` (not sure if we need to update this as this workgroup no longer exists?)
- `_posts/2019-10-31-sswg-update-2019.md`
- `_posts/2021-03-24-womens-history-month.md`

However, as we may not enforce the planned style guide for blog posts, I would only update references to workgroups (e.g. "Swift Server Work Group"), not the use of the noun "work group" (e.g. on [this line](https://github.com/apple/swift-org-website/blob/327234d321faa7db01f3d6e7e1edcc80369298a1/_posts/2019-10-31-sswg-update-2019.md?plain=1#L142)). Does this make sense?

> We should and `cc` the authors make them aware of the change.

Good idea. 👍 I will do this.

alexandersandberg commented:
Hey @twostraws, @tanner0101, @tomerd, @0xTim, and @devanshimodha, I just wanted to let you know that we have made some minor updates to your author bios in this PR to align the spelling of "workgroup" across the website (more info in this PR).

See the changes [here](https://github.com/apple/swift-org-website/pull/90/files#diff-fdca1c48b249c38bee14489176341e078b9cfcf168e19ba1c669b92fa7322c49).

No action is required on your part, but please let me know if you have any concerns. Thank you! 🙂

twostraws commented:
No concerns – thank you! 🙌

kaishin reviewed (approved):
Much nicer! Thank you.

tomerd commented:
@swift-ci test

shahmishal reviewed (approved):
LGTM! Thank you!
